server/server.cabal

```
@@ -31,7 +31,7 @@ executable server
 
   build-depends:
     base                  >= 4.13.0 && < 4.17,
-    aeson                 >= 1.4.7 && < 2.1,
+    aeson                 >= 1.4.7 && <= 2.1.0.0,
```

Use `<` constraints instead, so that we don't have to increase the bounds again when minor releases happen.

```suggestion
    aeson                 >= 1.4.7 && < 2.2,
```

---

ok - I was a bit worried that given we have only 2.1.0.0, it's hard to assure compat.

---

Yeah, not a given, but I think this is arguably the current standard for Haskell packages